README.md

```diff
@@ -38,17 +35,21 @@ Compared with other graph database solutions, **NebulaGraph** has the following
 * Horizontal scalability
 * Strong data consistency by RAFT protocol
 * OpenCypher-compatible query language
-* Role-based access control for higher level security
+* Role-based access control for higher-level security
 
 ## Notice of Release
 
-This repository hosts the source code of NebulaGraph versions before 2.0.0-alpha and after v2.5.x.  <br/>
-If you are looking to use the versions between v2.0.0 and v2.5.x, please head to [NebulaGraph repo](https://github.com/vesoft-inc/nebula-graph).
+NebulaGraph used to be split into three repositories: [Nebula-Graph](https://github.com/vesoft-inc/nebula-graph), [Nebula-Storage,](https://github.com/vesoft-inc/nebula-storage) and [Nebula-Common](https://github.com/vesoft-inc/nebula-common) for versions between v2.0.0 and v2.5.x, which will be archived.
+
+It's back to mono-repo since v2.6.0 and v3.x here at [github.com/vesoft-inc/nebula](https://github.com/vesoft-inc/nebula).
+
+Please check the latest released version via the documentation: https://docs.nebula-graph.io/.
```

Comment:
The word "latest" is an adjective and can be used to modify the verb "release". "the latest release" or "the latest version" would be enough.